lib/oneview-sdk/resource.rb

```diff
@@ -361,6 +361,12 @@ def recursive_like?(other, data = @data)
         return false unless data && data.respond_to?(:[])
         if val.is_a?(Hash)
           return false unless data.class == Hash && recursive_like?(val, data[key.to_s])
+        elsif val.is_a?(Array) && val.first.is_a?(Hash)
```

Comment:
With this code, the problem with ordered array just will occur when the arrays are of objects, if this is a problem for us, I can change the code =). When the array is of strings or numbers for example, the order is considered because the comparison occurs with `array == other_array`.
I like this idea of specifying the attibutes that are arrays ordered or not, because even when the array is of strings I can need compare without a specific order.